ShareCar.Api/ShareCar.Db/Repositories/User_Repository/UserRepository.cs

```diff
@@ -70,5 +71,92 @@ public async Task<IdentityResult> UpdateUserAsync(User user, ClaimsPrincipal pri
             var userAsync = await _userManager.UpdateAsync(_user);
             return userAsync;
         }
+
+        public UnauthorizedUser GetUnauthorizedUser(string email)
+        {
+            try
+            {
+                return _databaseContext.UnauthorizedUsers.Single(x => x.Email == email);
+            }
+            catch (Exception e)
+            {
+                return null;
+            }
+        }
+
+        public bool UserVerified(bool faceBookVerified, string loginEmail)
```

Comment:
bool fieldai/metodai turi prasidėt su `is` ar analogiškai. Siūlau pavadinimą `IsUserVerified`

Reply:
Siuo atveju bool reiskia (ne)pavykusiu operacija, Kai bus daromas error handling'as, turbut bool'as bus pakeistas i void'a